server/static/css/style.css

```diff
@@ -1,184 +1,167 @@
-html{
-    background-color:#fafafa;
-    color:#202020;
-    font-family: "Roboto";
+@import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
```

Comment:
Please do not use google fonts, as they are tracking the users and we would need a privacy statement (Because of GDPR) then

Reply:
Google fonts can be downloaded and used locally (Y)

Reply:
Changed to Tahoma, for better support for older browsers.